CMakeLists.txt

```diff
@@ -27,11 +27,12 @@ project(SFML VERSION 3.0.0)
 
 set(VERSION_IS_RELEASE OFF)
 
-# use ccache if available
-find_program(CCACHE_PROGRAM ccache)
-if(CCACHE_PROGRAM)
-    message(STATUS "Found ccache in ${CCACHE_PROGRAM}")
-    set_property(GLOBAL PROPERTY RULE_LAUNCH_COMPILE "${CCACHE_PROGRAM}")
+# Check if we are using any compiler launchers
+if(CMAKE_C_COMPILER_LAUNCHER)
+    message(STATUS "Using ${CMAKE_C_COMPILER_LAUNCHER} as C compiler launcher")
+endif()
+if(CMAKE_CXX_COMPILER_LAUNCHER)
+    message(STATUS "Using ${CMAKE_CXX_COMPILER_LAUNCHER} as C++ compiler launcher")
 endif()
```

Comment:
For what it's worth we can verify the use of ccache by looking at the compiler commands in CI or the compile_commands.json database.

<img width="329" alt="Screenshot 2023-11-03 at 10 29 12 AM" src="https://github.com/SFML/SFML/assets/39244355/2440f023-8f80-49a2-9822-00b4c773ca6f">

Reply:
This output isn't only for our CI.

Reply:
Right, that's why I also mentioned `compile_commands.json` which can be checked locally as well.